woocommerce/woocommerce-android
# [.github] Update issue templates

### Description

This PR is a result of editing / extending the issues templates through [GH issue template editor](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository). Some lines were changed by GH automatically (like removing `"` around strings), some were changed by me. What was changed by me:

1. `Bug` type of issue now contains `type: bug` label automatically.
2. `New Enhancement` type of issue now contains `type: enhancement` label automatically.
3. `Support` type of issue now contains `type: question` label automatically.
4. A new `Task` type of issue was added, with `type: task` default label.
5. A new `Technical Debt`  type of issue was added, with `type: technical debt` default label.

More context at p91TBi-aK9-p2#comment-11677

### To Test
- I see no ways to tests this other than going through the changes in the files.
- **Note: IINM, the `Danger - PR Check` [fails consistently](https://buildkite.com/automattic/woocommerce-android/builds/16399#018ba98f-bdf1-497f-9020-d6a94831421f/489-499) because PR has no changes from its POV.**
